arwen/elrondProtectedStorage.go

```diff
@@ -0,0 +1,7 @@
+package arwen
+
+// ElrondProtectedKeyPrefix prefixes all Elrond reserved storage. Only the protocol can write to keys starting with this.
+const ElrondProtectedKeyPrefix = "ELROND"
```

Comment:
this could come as input for arwen.

Reply:
as of protected functions - protected keys.

Reply:
Indeed. Created JIRA task for this.